recipes/rscape/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: rscape
+  version: 0.2.1
+
+source:
+  fn: rscape_v0.2.1.tar.gz
+  url: http://eddylab.org/software/rscape/rscape_v0.2.1.tar.gz
+  md5: 80cb9fe0e0a5ccc03473a5aa6eb7eada
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [not osx]
+    - llvm  # [osx]
+  run:
+    - libgcc    # [not osx]
+
+# fails for now due to a problem with R-scape dependency
+#test:
```

Comment:
There was a problem with parsing the `-h` argument in a script that is being called by R-scape. I added a test with a dummy Stockholm file. If the CI checks work, this should be a better test than just printing the help.